### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by remi-garcia**
I am checking for misspellings, using tools obviously. It is still in progress but I have a few questions:
- Should we use British English or American English? (e.g. for modelled/modeled)
- I replaced "arguement" with "argument" because I understood that the second is correct in British English and American English, is that right?

### CODE REVIEW

File: docs/src/tutorials/Getting started/an_introduction_to_julia.jl [hunk 1]
```diff
@@ -91,12 +91,12 @@ nothing #hide
...
 
 # !!! tip
-#     To make π (and most other greek letters), type `\pi` and then press
+#     To make π (and most other Greek letters), type `\pi` and then press
 #     `[TAB]`.
 
 typeof(π)
 
-# Athough if we do math with irrational numbers, they get converted to
+# Although if we do math with irrational numbers, they get converted to
```
Comment (remi-garcia) [inline docs/src/tutorials/Getting started/an_introduction_to_julia.jl:99]: I am not sure this sentence should start with "Although". I would rather use "However".

**Comment by mlubin** [comment]
This is very helpful, thanks!

> Should we use British English or American English?

American

File: docs/src/tutorials/Getting started/an_introduction_to_julia.jl (hunk 1 above)
Comment (odow) [reply docs/src/tutorials/Getting started/an_introduction_to_julia.jl:99]:
```suggestion
# However, if we do math with irrational numbers, they get converted to
```

**Comment by remi-garcia** [comment]
Checking is done. I also replaced some http links with https ones.

**Review by odow — APPROVED** [review]
Very useful! What tools did you use for this?

**Comment by remi-garcia** [comment]
Oh, nothing fancy. I just managed to have my spell check working in Atom (which was harder than it looks). Then it was mostly by hand.

Sorry if 
> using tools obviously

was misleading, it was to emphasize on the fact that despite my poor English corrections should be fine.

**Pull request merged**